<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LevelCreator</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="levelEgdeLabel">
        <property name="text">
         <string>Krawędź poziomu:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="levelEdgeSpinBox">
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>5000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="timeLimitLabel">
        <property name="text">
         <string>Limit czasowy (ms):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="timeLimitSpinBox">
        <property name="minimum">
         <number>60000</number>
        </property>
        <property name="maximum">
         <number>600000</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="zoomLabel">
        <property name="text">
         <string>zoom:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="zoomButtonUp">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="zoomButtonDown">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="buttonAddPlanet">
        <property name="text">
         <string>Dodaj planetę</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="asteroidBigAdd">
        <property name="text">
         <string>Dodaj dużą asteroidę</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="asteroidMediumAdd">
        <property name="text">
         <string>Dodaj średnią asteroidę</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="asteroidSmallAdd">
        <property name="text">
         <string>Dodaj małą asteroidę</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sellingPointAdd">
        <property name="text">
         <string>Dodaj punkt sprzedaży</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="spaceshipAdd">
        <property name="text">
         <string>Dodaj statek kosmiczny</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="playerLabel">
        <property name="text">
         <string>Statek gracza:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="playerComboBox"/>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="xPosLabel">
          <property name="text">
           <string>Pozycja x:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="xPosSpinBox">
          <property name="minimum">
           <double>-5000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>5000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="yPosLabel">
          <property name="text">
           <string>Pozycja y:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="yPosSpinBox">
          <property name="minimum">
           <double>-5000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>5000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="xVelocityLabel">
          <property name="text">
           <string>Prędkość x:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="xVelSpinBox">
          <property name="minimum">
           <double>-5000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>5000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="yVelocityLabel">
          <property name="text">
           <string>Prędkość y:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="yVelSpinBox">
          <property name="minimum">
           <double>-5000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>5000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="rotationLabel">
          <property name="text">
           <string>Rotacja:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="rotationSpinBox">
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>360.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="massLabel">
          <property name="text">
           <string>Masa:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="massSpinBox">
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>99999999999999999475366575191804932315794610450682175621941694731908308538307845136842752.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="2">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="resizeAnchor">
       <enum>QGraphicsView::NoAnchor</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>687</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Pl&amp;ik</string>
    </property>
    <addaction name="actionWczytaj"/>
    <addaction name="actionZapisz"/>
   </widget>
   <addaction name="menuPlik"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionWczytaj">
   <property name="text">
    <string>&amp;Wczytaj</string>
   </property>
  </action>
  <action name="actionZapisz">
   <property name="text">
    <string>&amp;Zapisz</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
